Code/src/main/java/nl/utwente/group10/ui/BackendUtils.java

```diff
@@ -31,4 +33,18 @@ public static Type dive(ConstT function, int steps) {
         }
         return function;
     }
+    
+    /**
+     * @return Whether or not t1 and t2 unify with each other.
```

Comment:
If you already have the inferred type here, that must mean that type inference has succeeded. In other words, if that's the case, this method could just be `return true;`.